src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException; // Правильний імпорт винятку
+import core.basesyntax.model.User;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RegistrationServiceTest {
+    private RegistrationService registrationService;
+    private StorageDaoImpl storage;
+
+    @Before
```

Comment:
Consider using `@BeforeAll` instead of `@Before` if the setup does not need to run before each test. `@BeforeAll` runs once before all tests, which can be more efficient if the setup is not modified by the tests.